api/lib/domain/models/CertificationCandidate.js

```diff
@@ -31,9 +31,12 @@ const certificationCandidateValidationJoiSchema_v1_5 = Joi.object({
   complementaryCertifications: Joi.array().required(),
   billingMode: Joi.when('$isCertificationBillingEnabled', {
     is: true,
-    then: Joi.string()
-      .valid(...Object.values(BILLING_MODES))
-      .required(),
+    then: Joi.when('$isSessionCertificationCenterScoNonManagingStudent', {
```

Comment:
On ne doit pas verifier pour tout les SCO (managing students ou non). Pour autant, on ne devrait pas tomber dans le cas du sco managing students
```suggestion
    then: Joi.when('$isSessionCertificationCenterSco', {
```